README.md

```diff
@@ -322,14 +311,19 @@ reregister all healthy services managed by marathon-consul to the new format. Un
 
 ## SSE Support
 
-In future callback interface between marathon and marathon-consul will be replaced by SSE.
 While using SSE please consider:
 - SSE is using Web module config for queues, event sizes, in the future will be moved to sse module,
```

Comment:
I prefer to keep this PR small. I'll move events configuration from `web` to `sse` or `events` in separate commit.